### PR TITLE
Turkish Translation of 2019 and 2020 Contributors Pages

### PR DESCRIPTION
**Pull request by fatmabadri**
Makes progress on #1044

### CODE REVIEW

File: src/templates/tr/2019/contributors.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "base/contributors.html" %}
+
+{% block title %}2019 Katkıda bulunanlar | HTTP Archive&#8217;dan Web Almanac{% endblock %}
+
+{% block description %}2019 Web Almanac&#8217;a Analistler, Yazarlar, Beyin Takımı, Tasarımcılar, Geliştiriciler, Editörler, İnceleyiciler ve Çevirmenler olarak katkıda bulunan {{ config.contributors.items() | length }} kişi.{% endblock %}
```
Comment (bazzadp) [inline src/templates/tr/2019/contributors.html:5]: Meta data should be pain text as it's not rendered as HTML so would prefer not to use smart quotes like `&#8217` if we can avoid it.

```suggestion
{% block title %}2019 Katkıda bulunanlar | HTTP Archive'dan Web Almanac{% endblock %}

{% block description %}2019 Web Almanac'a Analistler, Yazarlar, Beyin Takımı, Tasarımcılar, Geliştiriciler, Editörler, İnceleyiciler ve Çevirmenler olarak katkıda bulunan {{ config.contributors.items() | length }} kişi.{% endblock %}
```

File: src/templates/tr/2020/contributors.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends "base/contributors.html" %}
+
+{% block title %}{{ year }} Katkıda bulunanlar | HTTP Archive&#8217;dan Web Almanac{% endblock %}
+
+{% block description %}{{ year }}Web Almanac&#8217;a Analistler, Yazarlar, Tasarımcılar, Geliştiriciler, Editörler, Liderler, İnceleyiciler ve Çevirmenler olarak katkıda bulunan {{ config.contributors.items() | length }}kişi.{% endblock %}
```
Comment (bazzadp) [inline src/templates/tr/2020/contributors.html:5]:
```suggestion
{% block title %}{{ year }} Katkıda bulunanlar | HTTP Archive'dan Web Almanac{% endblock %}

{% block description %}{{ year }}Web Almanac'a Analistler, Yazarlar, Tasarımcılar, Geliştiriciler, Editörler, Liderler, İnceleyiciler ve Çevirmenler olarak katkıda bulunan {{ config.contributors.items() | length }}kişi.{% endblock %}
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks good, but suggested removing the entity coding from the meta data in case they cause problems.

**Review by fatmabadri — COMMENTED** [review]
remove the entity code from the meta data

**Pull request merged**